tests/phpunit/tests/post/thumbnails.php

```diff
@@ -408,6 +408,41 @@ public function test_post_thumbnail_size_filter_complex( $which_post, $expected
 		$this->assertSame( $expected, $result );
 	}
 
+	/**
+	 * @ticket 57490
+	 */
+	public function test_get_the_post_thumbnail_includes_loading_lazy() {
+		set_post_thumbnail( self::$post, self::$attachment_id );
+
+		$html = get_the_post_thumbnail( self::$post );
+		$this->assertStringContainsString( ' loading="lazy"', $html );
```

Comment:
```suggestion
		$this->assertStringContainsString( ' loading="lazy"', get_the_post_thumbnail( self::$post ) );
```

Reply:
A bit of a nit-pick, I think breaking this into separate lines makes it easier to read.

Reply:
Ok. Thanks! Please close this. 😄